barretenberg/cpp/src/barretenberg/op_queue/ecc_op_queue.hpp

```diff
@@ -307,7 +307,7 @@ class ECCOpQueue {
         Fr z_2 = 0;
         auto converted = scalar.from_montgomery_form();
         uint256_t converted_u256(scalar);
-        if (converted_u256.get_msb() <= 128) {
+        if (converted_u256.get_msb() < 128) {
```

Comment:
Could you please then remove the mitigation in the fuzzer and run it to make sure it doesn't crash?